introduction.md

```diff
@@ -0,0 +1 @@
+My name is Marta and I'm just following the course *Mastering Github* from *Code School*.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.